cfme/ansible/credentials.py

```diff
@@ -217,11 +222,11 @@ def _wait_until_appeared():
         return Credential(name, credential_type, appliance=self.appliance, **credentials)
 
 
-class Credential(Navigatable):
+class Credential(NavigatableMixin):
     """A class representing one Embedded Ansible credential in the UI."""
-    def __init__(self, name, credential_type, collection=None, appliance=None, **credentials):
-        self.collection = collection or CredentialsCollection(appliance=appliance)
-        Navigatable.__init__(self, appliance=self.collection.appliance)
+    def __init__(self, name, credential_type, collection, appliance=None, **credentials):
```

Comment:
**REQUIRED**
`appliance` argument is not used here anymore, so it should be removed.